### DISCUSSION TITLE
Copilot IDEA plugin conflicts with Docker plugin

### DISCUSSION THREAD

**Discussion by theheroonline** [community/community#112305] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When I am using the copilot IDEA plugin, it will scan the files occupying the target folder, causing conflicts with the Docker plugin in IDEA. This is an error message:Failed to deploy 'project Dockerfile: Dockerfile': java.io.IOException: 另一个程序已锁定文件的一部分，进程无法访问。

我在使用copilot IDEA插件时,copilot插件会扫描占用target文件夹里的文件,导致与IDEA里面的Docker插件冲突,这个是错误提示: Failed to deploy 'projectDockerfile: Dockerfile': java.io.IOException: 另一个程序已锁定文件的一部分，进程无法访问。

**Comment by zuoliang0** [comment]
I have the same problem.

**Comment by Wan9xy** [comment]
the same problem, and that happened at windows device, work well at my mac. but windows at my home, no fix, omg!!!

**Comment by xiwangly2** [comment]
I have the same problem.

**Comment by czl000** [comment]
I updated the plugin version and the problem disappeared
我更新了插件版本后问题消失
1.5.1.5285

**Reply by Wan9xy** [reply]
If the error happened when you install the latest version, try uninstall copilot first, and reinstall, it will fix the error.
